help/unsubscribe-from-a-stream.md

```diff
@@ -4,19 +4,44 @@ You can always unsubscribe from any stream in Zulip.
 
 {start_tabs}
 
+{tab|desktop-web}
+
 {!stream-actions.md!}
 
 1. Click **Unsubscribe**.
 
 {end_tabs}
 
-## From the streams page (alternate method)
+## Alternate method to unsubscribe from a stream
```

Comment:
This should be **Alternate methods for unsubscribe from a stream** since there are two below.